[docs] fix multiple things in the ip adapter guide (#6770)
huggingface/diffusers: docs/source/en/using-diffusers/loading_adapters.md

IP-Adapter works with most of our pipelines, including Stable Diffusion, Stable Diffusion XL (SDXL), ControlNet, T2I-Adapter, AnimateDiff.  And you can use any custom models finetuned from the same base models. It also works with LCM-Lora out of box.

You can find official IP-Adapter checkpoints in [h94/IP-Adapter](https://huggingface.co/h94/IP-Adapter).

IP-Adapter was contributed by [okotaku](https://github.com/okotaku).

> Let's first create a Stable Diffusion Pipeline. IP-Adapter relies on an image encoder to generate the image features, if your IP-Adapter weights folder contains a "image_encoder" subfolder, the image encoder will be automatically loaded and registered to the pipeline. Otherwise you can so load a [`CLIPVisionModelWithProjection`](https://huggingface.co/docs/transformers/main/en/model_doc/clip#transformers.CLIPVisionModelWithProjection) model and  pass it to a Stable Diffusion pipeline when you create it.

Reviewer: ```suggestion
Let's first create a Stable Diffusion Pipeline. IP-Adapter relies on an image encoder to generate the image features. If your IP-Adapter weights folder contains an "image_encoder" subfolder, the image encoder is automatically loaded and registered to the pipeline. Otherwise, you can load a [`~transformers.CLIPVisionModelWithProjection`] model and pass it to a Stable Diffusion pipeline when you create it.
```
